core/assets/wei.go

```diff
@@ -76,6 +76,11 @@ func NewWeiI[T constraints.Signed](i T) *Wei {
 	return NewWei(big.NewInt(int64(i)))
 }
 
+// Returns the Wei value of the input big.Int.
+func NewWeiString(i *big.Int) string {
```

Comment:
Funcs like this are typically called something like `FormatWei`

Reply:
thanks for the suggestion. Will update the name